Library/Formula/weechat.rb

```diff
@@ -14,6 +14,10 @@ class Weechat < Formula
 
   option "with-perl", "Build the perl module"
   option "with-ruby", "Build the ruby module"
+  option "with-python", "Build the python module"
```

Comment:
> That's why I added explanation for all of them to allow to compile with them

The process doesn't change for the end-user whether we add the explicit options or not - Currently if you want a "full" install of `weechat` you do `brew install weechat --with-perl --with-ruby --with-python --with-lua --with-guile --with-curl` - Nothing about that changes even if we make the `:optional` dependencies more explicit.

> By the way, what about building with them by default? 

We tend not to do this, because on a clean machine that would mean instead of getting 3 dependencies prior to being able to install `weechat` by default, you'd end up with 8, which is quite a bit more. We try to keep mandatory dependencies to a minimum unless there's significant popularity/demand for an option.